rep-0149.rst

```diff
@@ -1007,6 +1007,12 @@ Therefore the environment variable `ROS_VERSION` is used with the value being
 either `1` or `2`.
 This new environment variable can be defined beside the `ROS_DISTRO` one.
 
+An additional environment variable `ROS_PYTHON_VERSION` to choose the Python
+version in use (either `2` or `3`) is also provided.  In ROS 1, this allows
+users to test out packages on Python 3 before the distribution is officially
+switched to Python 3.  On ROS 2, changing this environment variable will have no
+effect, as it always uses Python 3.
```

Comment:
I don't think the last sentence is right. I would state that ROS 2 only supports Python 3. But if you do change the environment variable tools like `rosdep` will not work correctly anymore (for ROS 2).

Reply:
Right.  Fixed in b53636244c08faa785d2647f738a868c34317fbb.